lib/config-generator.js

```diff
@@ -141,34 +141,34 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
                 "browser,node": "globals: {...globals.browser, ...globals.node}"
             };
 
-            exportContent += `  {languageOptions: { ${envContent[this.answers.env.join(",")]} }},\n`;
+            exportContent += `  { files: ["**/*.js"], languageOptions: { ${envContent[this.answers.env.join(",")]} } },\n`;
         }
 
         if (this.answers.purpose === "syntax") {
 
             // no need to install any plugin
         } else if (this.answers.purpose === "problems") {
             this.result.devDependencies.push("@eslint/js");
-            importContent += "import pluginJs from \"@eslint/js\";\n";
-            exportContent += "  pluginJs.configs.recommended,\n";
+            importContent += "import js from \"@eslint/js\";\n";
+            exportContent += "  { files: [\"**/*.js\"], plugins: { js }, extends: [\"js/recommended\"] },\n";
```

Comment:
Similar to my previous comment, this should probably apply to more files (`.cjs`, `.mjs`, then `.jsx` if it's included in linting, etc.)